src/main/scala/org/ergoplatform/ErgoApp.scala

```diff
@@ -96,14 +102,12 @@ class ErgoApp(args: Args) extends ScorexLogging {
       None
     }
 
-
   private val syncTracker = ErgoSyncTracker(scorexSettings.network, timeProvider)
-  private val statsCollectorRef: ActorRef = ErgoStatsCollectorRef(readersHolderRef, networkControllerRef, syncTracker, ergoSettings, timeProvider)
```

Comment:
Like with ErgoNodeViewSynchronizer, we need to touch ErgoStatsCollectorRef to run it

Reply:
This variable is untouched by PR